apps/docs/pages/guides/auth/phone-login.mdx

```diff
@@ -82,37 +239,61 @@ To verify the one-time password (OTP) sent to the user's phone number, call [`ve
 >
 <TabPanel id="js" label="JavaScript">
 
+You should present a form to the user so they can input the 6 digit pin, then send it along with the phone number to `verifyOtp`:
+
 ```js
-const { data, error } = await supabase.auth.verifyOtp({
-  phone: '123456789',
+const {
+  data: { session },
+  error,
+} = await supabase.auth.verifyOtp({
+  phone: '+13334445555',
   token: '123456',
   type: 'sms'
 })
 ```
 
 </TabPanel>
-</Tabs>
+<TabPanel id="kotlin" label="Kotlin">
 
-## Set up a provider with Supabase Auth
+You should present a form to the user so they can input the 6 digit pin, then send it along with the phone number to `verifyPhoneOtp`:
 
-Supabase supports Phone Login with several communications platforms. Follow the guides below to set up a provider with Supabase Auth.
+```kotlin
+supabase.auth.verifyPhoneOtp(
+    type = OtpType.Phone.SMS,
+    phone = "+13334445555",
+    token = "123456"
+)
+```
 
-<div className="grid grid-cols-1 md:grid-cols-12 gap-10 not-prose py-8">
-  {PhoneLoginsItems.map((item) => (
-    <Link href={`${item.url}`} key={item.name} passHref className="col-span-4">
-      <IconPanel
-        title={item.name}
-        span="col-span-6"
-        icon={item.icon}
-        isDarkMode={item.isDarkMode}
-        hasLightIcon={item.hasLightIcon}
-      >
-        {item.linkDescription}
-      </IconPanel>
-    </Link>
-))}
+</TabPanel>
+<TabPanel id="http" label="HTTP">
 
-</div>
+```bash
+curl -X POST 'https://cvwawazfelidkloqmbma.supabase.co/auth/v1/verify' \
+-H "apikey: SUPABASE_KEY" \
+-H "Content-Type: application/json" \
+-d '{
+  "type": "sms",
+  "phone": "+13334445555",
+  "token": "123456"
+}'
+```
+
+</TabPanel>
+</Tabs>
+
+If successful the user will now be logged in and you should receive a valid session like:
+
+```json
+{
+  "access_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNjI3MjkxNTc3LCJzdWIiOiJmYTA2NTQ1Zi1kYmI1LTQxY2EtYjk1NC1kOGUyOTg4YzcxOTEiLCJlbWFpbCI6IiIsInBob25lIjoiNjU4NzUyMjAyOSIsImFwcF9tZXRhZGF0YSI6eyJwcm92aWRlciI6InBob25lIn0sInVzZXJfbWV0YWRhdGEiOnt9LCJyb2xlIjoiYXV0aGVudGljYXRlZCJ9.1BqRi0NbS_yr1f6hnr4q3s1ylMR3c1vkiJ4e_N55dhM",
+  "token_type": "bearer",
+  "expires_in": 3600,
+  "refresh_token": "LSp8LglPPvf0DxGMSj-vaQ"
+}
```

Comment:
hopefully these are mock values and don't actually contain information about a user / project

Reply:
mock I assume, I copied them over from the old guides, but good point, I'll swap out for placeholders